### ISSUE TITLE
Get App to pull well features based on a polygon.

### ISSUE THREAD

**Issue by micheal-w-wells** [bcgov/invasivesbc#711]
- [x] Get app to pull well features from a WFS service (DataBC) based on a polygon.  
- [ ] Work with Mike so that the fetch happens in the same cache data button/operation.  duplicate of #670 
- [x] Display wells 
will be done after #717:
- [ ] (and allow for toggle).

**Comment by micheal-w-wells** [comment]
The second bullet will be easy, just assume you're getting provided a polygon and I'll show you where to put it (the plan page will let the user draw one).
